sotopia/generation_utils/generate.py

```diff
@@ -58,6 +59,8 @@
     "human",
     "redis",
     "mistralai/Mixtral-8x7B-Instruct-v0.1",
+    "together_ai/togethercomputer/llama-2-7b-chat",
```

Comment:
Do we still need these models?

Reply:
No, but maybe it would still be helpful to have in documentation somewhere to show what's available. Or just point to LiteLLM docs?

Reply:
Can we make it backward compatible? For example, we can use LiteLLM as one kind of LLM_Name?